libs/profiles/src/profile.model.ts

```diff
@@ -42,37 +43,29 @@ export class Location {
 export class Address {
   @IsOptional()
   @IsString()
-  @ApiProperty()
   public raw: string;
   @IsOptional()
   @IsString()
-  @ApiProperty()
   public extra: string;
   @IsOptional()
   @IsString()
-  @ApiProperty()
   public postalCode: string;
   @IsOptional()
   @IsString()
-  @ApiProperty()
   public city: string;
   @IsOptional()
   @IsString()
-  @ApiProperty()
   public country?: string;
-  @ValidateNested()
+  @ValidateNested({ each: true })
```

Comment:
does it need @IsArray() ? same issue that we had in ocean?

Reply:
if its an array definitely